To: vim_dev@googlegroups.com
Subject: Patch 7.3.136
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.136
Problem:    Duplicate include of assert.h.
Solution:   Remove it.
Files:	    src/if_cscope.c


*** ../vim-7.3.135/src/if_cscope.c	2010-12-30 11:41:05.000000000 +0100
--- src/if_cscope.c	2011-03-03 15:01:15.000000000 +0100
***************
*** 13,19 ****
  
  #if defined(FEAT_CSCOPE) || defined(PROTO)
  
- #include <assert.h>
  #include <sys/types.h>
  #include <sys/stat.h>
  #if defined(UNIX)
--- 13,18 ----
*** ../vim-7.3.135/src/version.c	2011-02-25 18:38:29.000000000 +0100
--- src/version.c	2011-03-03 14:59:32.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     136,
  /**/

-- 
ARTHUR:  You fight with the strength of many men, Sir knight.
         I am Arthur, King of the Britons.  [pause]
         I seek the finest and the bravest knights in the land to join me
         in my Court of Camelot.  [pause]
         You have proved yourself worthy; will you join me?  [pause]
         You make me sad.  So be it.  Come, Patsy.
BLACK KNIGHT:  None shall pass.
                                  The Quest for the Holy Grail (Monty Python)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
